### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by haftamudesta**
This is my first hello microverse project which is implemented with HTML and CSS.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,18 +1 @@
...
-# Executables
-*.swf
-*.air
-*.ipa
-*.apk
-
-# Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
-# should NOT be excluded as they contain compiler settings and other important
-# information for Eclipse / Flash Builder.
+test.md
```
Comment (Ismailco) [inline .gitignore]: - [ ] In addition to test.md the `.gitignore` should contain the node_modules folder, so git wouldn't push the node modules to your Github repo, kindly add `node_modules` to your file

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
-# helloMicroverse
\ No newline at end of file
+# Project Name:hello microverse
```
Comment (Ismailco) [inline README.md]: - [ ] Pull Request should have a descriptive title telling you what changes are inside, as for now `feature` doesn't describe the changes inside the PR, kindly change it to a more meaningful title, (e.g. Setup Hello Microverse project)

Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] Good work adding a description that reflect the changes in your PR, however you can add more descriptive details

Comment (Ismailco) [inline README.md]: - [ ] The Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), as for now `feature` is not an action or feature, kindly in the future make sure you name your branch accordingly

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @haftamudesta,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @haftamudesta , 


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
 You should implement the changes requested by the previous reviewer 
- [ ] Pull Request should have a descriptive title telling you what changes are inside, as for now feature doesn't describe the changes inside the PR, kindly change it to a more meaningful title, (e.g. Setup Hello Microverse project)
- [ ]  Add node_modules in gitignore file. 
- [ ] Your branch name should be have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**